lib/i18n/tests/interpolation.rb

```diff
@@ -54,6 +54,11 @@ module Interpolation
         assert_equal 'Hi Yehuda!', interpolate(:interpolate, :name => 'Yehuda')
       end
 
+      test "interpolation: given an array interpolates each element" do
+        I18n.backend.store_translations(:en, :array_interpolate => ['Hi', '%{name}'])
+        assert_equal ['Hi', 'Bartuz'], interpolate(:array_interpolate, :name => 'Bartuz')
+      end
```

Comment:
IMO this test doesn't actually fully cover the use case since there's only one string that's being interpolated. You should add a second string that includes the `%{name}` placeholder like `Hello %{name}` and make sure it interpolates correctly.